python/src/main/parser.py

```diff
@@ -10,3 +10,7 @@ def create_person(row):
 def create_item(row):
     person_email, name, price, priority, hyperlink = row[0], row[1], row[2], row[3], row[4]
     return Item(person_email, name, price, priority, hyperlink)
+
+def parse_list_of_items(lst):
+    for i in range(0, len(lst), 5):
+        yield lst[i: i+5]
```

Comment:
Do you mean to use yield here or return?
Do you know the difference?

Reply:
I meant to use yield because when I used return for some reason it would only ever return the first list (the first 5 elements in a list) and not the rest, whereas yield would let me return all of them.
I think I know the difference - though I may be blind to some of the pitfalls of using yield.